### PR TITLE
updates to the iQTS page

### PR DESCRIPTION
**Pull request by Sarah-DfE**
### Trello card

https://trello.com/c/ngQKGS9u/6946-updates-to-the-iqts-page-from-itru-team

### Context

Small updates to the page from the ITRU team

### CODE REVIEW

**Review by joegibb — APPROVED** [review]
Looks good to me

File: app/views/content/non-uk-teachers/international-qualified-teacher-status.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ If you’re not sure whether your qualifications meet the entry requirements for
 
 Fees range from $fees_iQTS_low$ to $fees_iQTS_high$ for a one-year, full-time iQTS course. Check training provider websites for full details.
 
-Applicants for iQTS are not eligible for loans, bursaries or any other type of financial support from Student Finance England to help cover the costs of studying iQTS.
+Applicants for iQTS are not eligible for loans, bursaries or any other type of financial support from the Department for Education and Student Finance England to help cover the costs of studying iQTS.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/international-qualified-teacher-status.md:70]:
```suggestion
Applicants for iQTS are not eligible for loans, bursaries or any other type of financial support from the Department for Education and Student Finance England to help cover the costs of studying for iQTS.
```

**Comment by gemmadallmandfe** [comment]
@Sarah-DfE This section feels like it needs updating - we are already in academic year 2024/25 - should they be in the standard list rather than at the end?

image


Can the Chichester link go to the iQTS page (would need to confirm this): 
https://www.chi.ac.uk/teaching/course/ipgce/

**Comment by gemmadallmandfe** [comment]
@Sarah-DfE Could we possibly remove the promo (content/non-uk-teachers/promos/events-promo-iqts) from this page (and delete the promo file as well) - it's not really appropriate as we don't have many online GIT events at present

We don't need to replace it with any other promos (we don't have them generally on other non-UK pages) :-)

**Pull request merged**